integration-test-groups/azure/azure-resources.sh

```diff
@@ -103,7 +110,8 @@ function createResources() {
     echo 'export AZURE_CLIENT_SECRET="'${AZURE_CLIENT_SECRET}'"'
     echo 'export AZURE_TENANT_ID="'${AZURE_TENANT_ID}'"'
     echo 'export AZURE_VAULT_NAME="'${AZURE_VAULT_NAME}'"'
-    echo
+    echo 'export AZURE_VAULT_EVENT_HUBS_BLOB_CONTAINER_NAME="'${AZURE_BLOB_CONTAINER_NAME}'"'
```

Comment:
```suggestion
    echo 'export AZURE_VAULT_EVENT_HUBS_BLOB_CONTAINER_NAME="'${AZURE_VAULT_BLOB_CONTAINER_NAME}'"'
```

Reply:
Thanks for that. I fixed the name in the key-vault module's script, but evidently did not propagate it here.

Reply:
fixed